lib/list.c

```diff
@@ -47,6 +47,7 @@ M0_EXPORTED(m0_list_is_empty);
 
 M0_INTERNAL bool m0_list_link_invariant(const struct m0_list_link *link)
 {
+#if 0
```

Comment:
Please revert changes from list.c and tlist.c. We can disable expensive invariants at compile time.

Reply:
ok.